include/openrave/kinbody.h

```diff
@@ -330,6 +332,39 @@ class OPENRAVE_API KinBody : public InterfaceBase
         bool _bVisible = true; ///< if true, geometry is visible as part of the 3d model (default is true)
         bool _bModifiable = true; ///< if true, object geometry can be dynamically modified (default is true)
 
+        struct CalibrationBoardParams { ///< used by GT_CalibrationBoard
+            int numDotsX; ///< number of dots in x direction, minimum 3
+            int numDotsY; ///< number of dots in y direction, minimum 3
+            float dotsDistanceX; ///< distance between center of dots in x direction
```

Comment:
Is there a particular reason why `float` is used instead of `dReal`? Same for `RaveVector`